### ISSUE TITLE
Build error on Xcode 14.3 (RNSSAddToSiriButtonViewManager.m new is unavailable)

### ISSUE THREAD

**Issue by JakobLierman** [Gustash/react-native-siri-shortcut#131]
Xcode version: `14.3`
React Native version: `0.68.5`
Package version: `3.2.2`

Xcode throws following error whilst trying to start the application.
`.../node_modules/react-native-siri-shortcut/ios/RNSSAddToSiriButtonViewManager.m:45:70 'new' is unavailable`

**Comment by Arkkeeper** [comment]
I've submitted a PR addressing this
https://github.com/Gustash/react-native-siri-shortcut/pull/132

**Comment by hakansaglam29** [comment]
I am getting also the same error after upgrading Xcode version 14.3.

**Comment by Gustash** [comment]
Sorry guys, my personal life has been a bit rocky as of late. I'll try to have a look at it this week, apologies for the delay

**Comment by gavrichards** [comment]
I can confirm https://github.com/Gustash/react-native-siri-shortcut/pull/132 fixes the issue, I'm applying the fix using [patch-package](https://github.com/ds300/patch-package) and rolling into production. Thanks!

**Comment by gaberogan** [comment]
My team was looking at this issue and noted that 

```objc
INUIAddVoiceShortcutButton *button = [INUIAddVoiceShortcutButton alloc];
```

can cause undefined behavior without init, and that Apple wants you to use `initWithStyle` like so:

```objc
INUIAddVoiceShortcutButton *button = [[INUIAddVoiceShortcutButton alloc] initWithStyle:INUIAddVoiceShortcutButtonStyleAutomatic];
```

**Comment by Gustash** [comment]
Fixed in v3.2.3
